cribl_stream/README.md

```diff
@@ -0,0 +1,72 @@
+# Cribl Stream
+
+## Overview
+[Cribl Stream][1] helps you process machine data – logs, instrumentation data, application data, metrics, and so on – in real time, and deliver it to your analysis platform of choice. It allows you to:
+
+- Add context to your data, by enriching it with information from external data sources.
+- Help secure your data, by redacting, obfuscating, or encrypting sensitive fields.
+- Optimize your data, per your performance and cost requirements.
```

Comment:
The overview section should include: 

1. What Cribl does in 1-2 sentences 
2. What will customers get from this integration (This should be clear to mutual customers but also potential new customers) 
3. Types of data being sent to Datadog

Reply:
This section seems to have the appropriate information to satisfy the requirements. If additional information is needed, please let us know and we will respond.